public/modules/custom/helfi_ahjo/helfi_ahjo.module

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * Implements hook_theme().
+ */
+function helfi_ahjo_theme($existing, $type, $theme, $path) {
```

Comment:
after doing all the cleanup and comments, run phpcs and fix all condign standards, comments, dep injection etc...